structuredheadings/plugin.js

```diff
@@ -12,6 +12,8 @@
     pre: new CKEDITOR.style({ element: "pre" })
   };
 
+  var listNodeNames = { ol: 1, ul: 1 };
```

Comment:
It was my preferred way to pass queries into path.contains in the past (maybe more complex queries)

That said, looking at the code http://docs.ckeditor.com/source/elementpath.html#CKEDITOR-dom-elementPath-method-contains i apparently can also pass this in as an array